src/wasm/wasm-emscripten.cpp

```diff
@@ -833,7 +849,9 @@ std::string EmscriptenGlueGenerator::generateEmscriptenMetadata(
   meta << "  \"externs\": [";
   commaFirst = true;
   ModuleUtils::iterImportedGlobals(wasm, [&](Global* import) {
-    meta << nextElement() << "\"_" << import->base.str << '"';
+    if (!(import->module == ENV && import->name == STACK_INIT)) {
+      meta << nextElement() << "\"_" << import->base.str << '"';
+    }
```

Comment:
What happens if you don't do this?

Reply:
Without this it would say that there is an extern "_STACKTOP". We don't need that since the special imports are already set up. Also this would be wrong with the `_` prefix, so we'd need to fix that if we needed it (but we don't luckily).